website/pages/gamejam_se.md

```diff
@@ -1,6 +1,22 @@
 <img src="/static/img/logga.png" alt="LiU Game Jam" id="gamejam-logo">
-<h1>MicroJam 2020</h1>
-Gör dig redo för vårt uppkommande evenemang [Microjam](/microjam/se/)!
+<h1>Fall Game Jam 2020</h1>
+Gör dig redo för vårt kommande evenemang Fall Game Jam!
+Den 13:e - 15:e november hålls årets upplaga av Fall Game Jam.
+På grund av att COVID-19 fortfarande orsakar förödelser i världen hålls jammet på distans via vår [Discord-server](https://discord.gg/YXV6dpc). Kolla även in på vårt [Facebook-event](https://www.facebook.com/events/4918751154809003/).
```

Comment:
"Kolla in på vårt Facebook-event" kombinerar två uttryck, vi vill ha någon av
- "kolla in vårt Facebook-event" (vilket jag tycker börja närma sig anglicism men jag vet inte, det kanske går att säga så)
- "ta även en titt på vårt Facebook-event"
- "kolla även på vårt Facebook-event"

Jag gillar den sista mest tror jag men jag har tittat för mycket på den här meningen så jag vet inte längre vad ord är.

Reply:
Jag tror andra förslaget låter bäst, sista känner jag har samma "tvingande" ton som det som står just nu vilket är lite offputting.